Authorization/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,100 @@
+<!--        <preConditions onFail="CONTINUE">-->
+<!--            <not>-->
+<!--                <tableExists tableName="role"/>-->
+<!--            </not>-->
+<!--        </preConditions>-->
+<!--        <createTable tableName="role">-->
+<!--            <column name="id" type="int">-->
+<!--                <constraints primaryKey="true" uniqueConstraintName="role_id_unq"/>-->
+<!--            </column>-->
+<!--            <column name="name" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="role_name_unq"/>-->
+<!--            </column>-->
+<!--        </createTable>-->
+<!--        <addForeignKeyConstraint baseTableName="role" baseColumnNames="id" constraintName="role_id_fk"-->
+<!--                                 referencedTableName="users_roles" referencedColumnNames="role_id"/>-->
+<!--    </changeSet>-->
+
+<!--    &lt;!&ndash;    Create the table user if does not exist&ndash;&gt;-->
+<!--    <changeSet id="3" author="Nikolay3">-->
+<!--        <preConditions onFail="CONTINUE">-->
+<!--            <not>-->
+<!--                <tableExists tableName="user"/>-->
+<!--            </not>-->
+<!--        </preConditions>-->
+<!--        <createTable tableName="user">-->
+<!--            <column name="id" type="int">-->
+<!--                <constraints primaryKey="true" primaryKeyName="id"/>-->
+<!--            </column>-->
+<!--            <column name="username" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="username_unq"/>-->
+<!--            </column>-->
+<!--            <column name="password" type="varchar(250)"/>-->
+<!--            <column name="email" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="email_unq"/>-->
+<!--            </column>-->
+<!--            <column name="refresh_token" type="varchar(250)">-->
+<!--                <constraints unique="true" uniqueConstraintName="refresh_token_unq"/>-->
+<!--            </column>-->
+<!--        </createTable>-->
+<!--        <addAutoIncrement tableName="user" columnName="id" columnDataType="int"/>-->
+<!--        <addForeignKeyConstraint baseTableName="user" baseColumnNames="id" constraintName="user_id_fk"-->
+<!--                                 referencedTableName="users_roles" referencedColumnNames="user_id"/>-->
+
+        <!--        If the table user is empty, then inserts admin and this admin will be added in users_roles table -->
+<!--    </changeSet>-->
+    <changeSet id="4" author="Nikolay4">
+        <preConditions onFail="CONTINUE">
+            <tableIsEmpty tableName="user"/>
+        </preConditions>
+        <insert tableName="user">
+            <column name="id" value="1"/>
+            <column name="username" value="admin"/>
+            <column name="password" value="12345"/>
+            <column name="email" value="admin@mail.com"/>
+        </insert>
+        <insert tableName="users_roles">
```

Comment:
Foreign consistency checks are broken.

assignment to role 1 is inserted here, but role 1 doesn't exist.
The proper order is first to insert into `role`, and only after that assign roles.